m3u8/model.py

```diff
@@ -861,6 +861,9 @@ def __str__(self):
         if self.iframe_stream_info.bandwidth:
             iframe_stream_inf.append('BANDWIDTH=%d' %
                                      self.iframe_stream_info.bandwidth)
+        if self.iframe_stream_info.average_bandwidth:
+            iframe_stream_inf.append('AVERAGE-BANDWIDTH=%d' %
```

Comment:
Can you please add a test for this code? I can see a typo here `iframe_stream_inf`. It should be `iframe_stream_info`, right?

Reply:
Thank you @mauricioabreu for your feedback. Interestingly, if `iframe_stream_inf` is a typo, then it is one I have copied from line 862. The pre-existing code also referred to `iframe_stream_inf` and not to `iframe_stream_info`. I will have a look at it later, just in case the typo is the official way forward.

Happy to add a test.

Reply:
@mauricioabreu, I had a look at git blame going back 9 years and it seems to me that `iframe_stream_inf` is how it has been spelled ever since, meaning that the code above has always referred to 'self.iframe_stream_info.xyz` yet the `append` is being applied to `iframe_stream_inf`.

Do you think the remainder of the code that has, I assume, been in use for a while, needs changing? I wonder come it has not been noticed earlier.

Reply:
You are right @RafalLukawiecki sorry!

Could you please add a test then? Thank you for your contribution! (after the test is added I will merge it)

Reply:
I have added a couple of tests, both to the parser and to the generator. As a result, I have also added the correct type information for the average_bandwidth attribute to `parser.py`. Hopefully this helps, let me know if you have any other suggestions.